lib/ex_unit/lib/ex_unit/runner.ex

```diff
@@ -5,45 +5,46 @@ defmodule ExUnit.Runner do
 
   @rand_algorithm :exs1024
 
-  def run(opts, load_us) do
-    {:ok, manager} = EM.start_link()
-    {:ok, stats} = EM.add_handler(manager, ExUnit.RunnerStats, opts)
-    {opts, config} = configure(manager, opts)
+  @type manager :: ExUnit.EventManager.manager()
+
+  def run(load_time, opts) when (is_integer(load_time) or is_nil(load_time)) and is_list(opts) do
```

Comment:
Not sure if its a wise choice to change the argument order here for the sake of consistency.
Could it affect third party libraries? 
WDYT @josevalim?

Reply:
Let's please just avoid those changes generally. The benefits are very little over the history change and potential breakage.